query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala

```
@@ -410,7 +410,10 @@ class QueryResource(secondary: Secondary,
   private def parseFuseColumnMap(s: String): Map[String, String] = {
     s.split(';')
      .map { item => item.split(',') }
-     .map { case Array(a, b) => (a, b) }
+     .map { case Array(a, b) => (a, b)
+            case _ =>
```

Maybe also throw the actual exception too.

---

Or include the message from it on the new one.

---

The actual exception is no match case in scala